docs/open-source/index.mdx

```diff
@@ -1,8 +1,18 @@
 ---
 title: Open source
-description: TODO - filler text so that the SEO check doesn't complain!
+description: Qiskit open-source introduction
 ---
 
 # Open source
 
-Blah blah
+Since 2016, IBM Quantum has harnessed the power of community-driven innovation by making quantum computing resources accessible to everyone via the cloud.
+This groundbreaking initiative includes open-sourcing key parts of the quantum software stack and introducing Qiskit, a versatile quantum computing framework.
+Today, Qiskit is the most popular quantum framework, thanks to its vibrant community.
+The Qiskit community is a dynamic and welcoming group of developers, scientists, and enthusiasts collaborating to advance software, research, education, and business through quantum computing.
+By making quantum computing open and accessible, we accelerate innovation and discover new applications for this transformative technology.
+
+As quantum technology evolves, contributions from the open-source Qiskit Ecosystem projects and developers are crucial in shaping its future.
+The development of a dense ecosystem is vital not only for extending Qiskit's capabilities but also for spreading involvement and engagement across various sectors.
```

Comment:
I think this sentence about the various sectors an be dropped, it feels a bit vague and i think the other sentences put the main point across better